deploy-0chain/action.yml

```diff
@@ -379,20 +379,7 @@ runs:
         timeout 300 ./stake_blobbers.sh
         if [[ $? -ne 0 ]]
           then
-            cat build.log
-            cd ..
-            echo '#!/bin/bash
-            RED="\033[0;31m"
-            NC="\033[0m"
-            echo
-            echo -e "${RED}======================================================${NC}"
-            echo -e "${RED}RETRYING DEPLOYMENT ($(echo $1 | awk '"'"'{ print toupper($0) }'"'"') ATTEMPT):${NC}"
-            echo -e "${RED}======================================================${NC}"
-            echo
-            ./teardown.sh && ./deploy.sh && cd $GITHUB_WORKSPACE && ./stake_blobbers.sh $1;
-            ' > ./retry.sh;
```

Comment:
This retry logic is important for stable CI 
We shouldn't be removing it

Reply:
fixed